data_rows/QueryHistory.js

```diff
@@ -0,0 +1,147 @@
+import uuid4 from "uuid/v4";
+
+import DataRow from "./DataRow";
+import { LoginHistory } from "./LoginHistory";
```

Comment:
if I'm only exporting one thing from a module, I like to use export default, then you don't have to put braces around it like this.

Reply:
when I switched to using the "package" (./index.js) I got errors and thought I had to remove the default. But that's not the case, not sure what I was doing wrong but I'll go back to defaults